src/library_exceptions.js

```diff
@@ -242,7 +242,11 @@ var LibraryExceptions = {
   // We'll do that here, instead, to keep things simpler.
   __cxa_find_matching_catch__deps: ['$exceptionLast', '$ExceptionInfo', '__resumeException', '__cxa_can_catch', 'setTempRet0'],
   __cxa_find_matching_catch: function() {
-    var thrown = 
+    // Here we use explicit calls to `from64`/`to64` rather then using the
+    // `__sig` magic.  This is because the `__sig` wrapper uses arrow function
+    // notation which is not compatible with the use of `arguments` in this
+    // function.
```

Comment:
What is `__sig` magic?

Reply:
`p` argument in the `__sig` signature are automatically converted from bitint and `p` return value are automatically converted back.  This is how emscripten avoids dealing with bigint values for pointers under wasm64.. it just coverts them to i53 numbers.